doc/api/errors.md

```diff
@@ -221,8 +221,7 @@ Error.captureStackTrace(myObject);
 myObject.stack;  // similar to `new Error().stack`
 ```
 
-The first line of the trace, instead of being prefixed with `ErrorType:
-message`, will be the result of calling `targetObject.toString()`.
+The first line of the trace will be prefixed with `ErrorType.name: message`.
```

Comment:
Agreed.
I'd assume this paragraph and the snippet above it have "evolved" separately and have become disjoint.
the actual formula is:
```js
`${myObject.name || 'Error'}${String(myObject.message) ? (': ' + String(myObject.message)) : ''}`
```
But IMHO ``` `${myObject.name}: ${myObject.message}` ``` is good enough